dvc/commands/experiments/apply.py

```diff
@@ -33,10 +25,11 @@ def add_parser(experiments_subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     experiments_apply_parser.add_argument(
```

Comment:
[Q] Is it ok to deprecate this now? It has not been working for a while.

Reply:
Yep, I think it's fine